scripts/configure_dev.sh

```diff
@@ -1,15 +1,41 @@
 #!/usr/bin/env bash
 set -e
 
+HELP="Usage: $0 [-s]
+Installs host level dependencies necessary to build go-algorand.
+
+Options:
+    -s        Skips installing go dependencies
+    -f        Force dependencies to be installed (May overwrite existing files)
+"
+
+while getopts ":sfh" opt; do
+  case ${opt} in
+    s ) SKIP_GO_DEPS="True"
+      ;;
+    f ) FORCE="True"
+      ;;
+    h ) echo "${HELP}"
+        exit 0
+      ;;
+    \? ) echo "${HELP}"
+        exit 2
+      ;;
+  esac
+done
+
 SCRIPTPATH="$( cd "$(dirname "$0")" ; pwd -P )"
```

Comment:
Is this needed?  Is this ever invoked directly from the cli rather than through scripting?